File: kata-6kyu/FizzBuzz-backwards.js
function reverseFizzBuzz(arr) {
  let fizz = [];
  let buzz = [];
  for(let i = 0 ; i < arr.length; i++){
    if(arr[i] === 'Fizz' || arr[i] === 'FizzBuzz')  fizz.push(i+1);
    if(arr[i] === 'Buzz' || arr[i] === 'FizzBuzz')  buzz.push(i+1);
  }
  return [fizz[0],buzz[0]]
}

File: python-kata-6kyu/FizzBuzz-backwards.py
def reverse_fizz_buzz(array):
    fizz = []
    buzz = []
    for i in range(0,len(array)):
        if array[i] in ['Fizz','FizzBuzz'] : fizz.append(i+1)
        if array[i] in ['Buzz','FizzBuzz'] : buzz.append(i+1)
    return (fizz[0],buzz[0])
